src/prefect/serialization/schedule.py

```diff
@@ -39,6 +39,26 @@ class Meta:
     end_date = DateTimeTZ(allow_none=True)
     interval = fields.TimeDelta(precision="microseconds", required=True)
 
+    @post_dump
+    def _interval_validation(self, data: dict, **kwargs: Any) -> dict:
+        """
+        Ensures interval is at least one minute in length
+        """
+        if data["interval"] / 1e6 < 60:
+            raise ValueError(
+                "Interval can not be less than one minute when deploying to Prefect Cloud."
+            )
+        return data
+
+    @post_load
+    def create_object(self, data: dict, **kwargs: Any):
+        if data["interval"].total_seconds() < 60:
```

Comment:
@jlowin can you sanity check me here? I was surprised to find this was a `timedelta` object in my tests instead of a raw number and want to make sure this won't cause any errors if the deserialization occurs in another order -- should I perform a type check here?

Reply:
I think you're looking good -- `post_load` is called *after* the deserialization takes place, so it should be a full Python object.